sources/packages/backend/apps/api/src/route-controllers/application/models/application.dto.ts

```diff
@@ -73,17 +73,21 @@ export interface ApplicationFormData extends ApplicationData {
   };
 }
 /**
- * Application DTO with primary identifier(s)
+ * Application and program year base
+ * information.
  */
-export class ApplicationIdentifiersAPIOutDTO {
+export class ApplicationProgramYearAPIOutDTO {
   id: number;
   applicationNumber: string;
+  programYear: string;
 }
```

Comment:
Because `ApplicationIdentifierAPIOutDTO` is only used in a few places like `ApplicationBaseAPIOutDTO`, I think it's safe to rename it to `ApplicationProgramYearAPIOutDTO`. As per our discussion,  I think we can use another class like `ApplicationAppealAPIOutDTO` which extends `ApplicationIdentifierAPIOutDTO` and use it to send data from the backend to frontend and form.io; otherwise, I think we could rename it `ApplicationProgramYearAPIOutDTO` to `ApplicationAppealAPIOutDTO` or something that's more relevant to the API endpoint for student appeals.

Reply:
@lewischen-aot Initially I thought it is a good idea to either extend and create or create from scratch `ApplicationAppealAPIOutDTO` but then I remembered why I wasn't creating this DTO. 

The reason was there is no single property inside the DTO which is appeal related (e.g. appeal id or appeal status) and hence I want to keep it the way it is now.